src/composables/app-can-wrapper/use-app-can-wrapper.ts

```diff
@@ -72,13 +77,18 @@ export function useAppCanWrapper ({ store }: UseAppCanWrapperParam) {
     for (const entity in normalizedParamsPayload) {
       const { action } = normalizedParamsPayload[entity]
 
+      /**
+       * @example action: ['list', 'show']
+       */
+      const normalizedAction = Array.isArray(action) ? action : [action]
```

Comment:
se for trocar, trocar tudo pra `actions`

Reply:
Alinhado disc.